components/inputs/input-date.js

```diff
@@ -20,7 +23,7 @@ export function formatISODateInUserCalDescriptor(val) {
  * A component that consists of a text input field for typing a date and an attached calendar (d2l-calendar) dropdown. It displays the "value" if one is specified, or a placeholder if not, and reflects the selected value when one is selected in the calendar or entered in the text input.
  * @fires change - Dispatched when a date is selected or typed. "value" reflects the selected value and is in ISO 8601 calendar date format ("YYYY-MM-DD").
  */
-class InputDate extends LocalizeCoreElement(LitElement) {
+class InputDate extends FormElementMixin(LocalizeCoreElement(LitElement)) {
```

Comment:
I used `FormElementMixin` in with input-date as well as validation-custom. Let me know if I used these incorrectly. I had added a console log within form element mixin locally to confirm the errors were happening when expected, so it seems to work.

Reply:
`FormElementMixin` does support `d2l-validation-custom` so there's no issue with that. Although, I think in this case we can likely accomplish validation without it since its only needed for `async` validation.